README.md

```diff
@@ -1,21 +1,28 @@
 ## Token Merchant Checkout Sample: Java
 
+This sample code shows how to enable the Token Merchant Checkout
+experience on a simple web server.
+
+To build this code, you need Java Development Kit (JDK) version 8 or later.
+
 **To create a member:**
 
-* Edit `setup/src/main/java/io/token/setup/Application.java` and replace the
-  username `marianoTest5` with another alphanumeric string.
+* Choose a username, an alphanumeric string.
 * Compile: `./gradlew shadowJar`
-* Run the setup utility: `java -jar setup/build/libs/setup-1.0.1-all.jar`
+* Run the setup utility: `java -jar setup/build/libs/setup-1.0.1-all.jar merchantTest4`,
```

Comment:
Can we just create keys if there are none? Then you don't need to have 2 steps. Then to reset you delete your keys directory.